http/mixed-content.json

```
@@ -0,0 +1,322 @@
+{
+  "http": {
+    "mixed_content": {
+      "__compat": {
+        "description": "Blocks some or all insecure mixed content",
+        "mdn_url": "https://developer.mozilla.org/docs/Web/Security/Mixed_content",
+        "spec_url": "https://www.w3.org/TR/mixed-content/#intro",
+        "support": {
+          "chrome": {
+            "version_added": "79"
+          },
+          "chrome_android": "mirror",
+          "edge": "mirror",
+          "firefox": {
+            "version_added": "23"
+          },
+          "firefox_android": "mirror",
+          "ie": {
+            "version_added": false
+          },
+          "oculus": "mirror",
+          "opera": "mirror",
+          "opera_android": "mirror",
+          "safari": {
+            "version_added": true
+          },
+          "safari_ios": "mirror",
+          "samsunginternet_android": "mirror",
+          "webview_android": "mirror"
+        },
+        "status": {
+          "experimental": false,
+          "standard_track": true,
+          "deprecated": false
+        }
+      },
+      "allow_file_urls": {
+        "__compat": {
+          "description": "Allow mixed content from <code>file:</code> URLs",
+          "spec_url": "https://www.w3.org/TR/mixed-content/#intro",
+          "support": {
+            "chrome": {
+              "version_added": true
+            },
+            "chrome_android": "mirror",
+            "edge": "mirror",
+            "firefox": {
+              "version_added": true
```

I "think" this is from day one but not certain. Should I put "23" (first version)? Ditto for the other subfeatures.

---

Let's try `≤` here too

```suggestion
              "version_added": "≤23"
```